README.md

```diff
@@ -79,17 +79,57 @@ EOS
 
 ## Example Scenario
 
-### What the Cluster Admin does
+In this example there a are 2 roles:
 
-* A cluster admin installs into the cluster a Backing Service Operator that is "bind-able," in other
-  words a Backing Service Operator that exposes binding information in secrets, status, and/or spec
-  attributes. The Backing Service Operator may represent a database or other services required by
-  applications. We'll use [postgresql-operator](https://github.com/baijum/postgresql-operator) to
-  demonstrate a sample use case.
+* Cluster Admin - Installs the operators to the cluster
+* Application Developer - Imports a Node.js application, creates a DB instance, creates a request to bind the application and DB (to connect the DB and the application).
 
-* The cluster-admin installs the database operator using an `OperatorSource`:
+### Cluster Admin
 
-``` yaml
+The cluster admin needs to install 2 operators into the cluster:
+
+* Service Binding Operator
+* Backing Service Operator
+
+A Backing Service Operator that is "bind-able," in other
+words a Backing Service Operator that exposes binding information in secrets, config maps,status, and/or spec
+attributes. The Backing Service Operator may represent a database or other services required by
+applications. We'll use [postgresql-operator](https://github.com/baijum/postgresql-operator) to
+demonstrate a sample use case.
+
+#### Install the Service Binding Operator using and `OperatorSource`:
```

Comment:
should be:   using **an** 'OperatorSource'

Reply:
Thanks @ldimaggi, Fixed